style.css

```diff
@@ -0,0 +1,704 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap');
+
+@media (min-width: 320px) {
+  body {
+    margin: 0;
+    overflow-x: hidden;
+  }
+  .container { margin: 52px 24px 0 24px; }
+
+  header {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+  }
+  header ul { display: none; }
+
+  header a {
+    /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+    font-family: 'Poppins';
+    text-decoration: none;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    color: #6070ff;
+  }
+
+  .flex-nav { color: #6070ff; }
+
+  .headline {
+    background-image: url(./assets/headershapesmobile.png);
```

Comment:
- Kindly check to be sure that you have this in the folder. If you don't, like I think you don't, try downloading it or rename the file.